Header header
geometry_msgs/PoseStamped[] poses
float64[] risks